### PR TITLE
Fix up samples projects

### PR DESCRIPTION
**Pull request by drwill-ms**
Sort/remove usings
Format csproj's (replace tabs with spaces)
Fix broken references

### CODE REVIEW

File: iothub/device/samples/getting started/FileUploadSample/FileUploadSample.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (drwill-ms) [inline iothub/device/samples/getting started/FileUploadSample/FileUploadSample.csproj:12]: Json references were failing in debug (at least)

File: iothub/device/samples/getting started/FileUploadSample/FileUploadSample.csproj [hunk 2]
```diff
@@ -21,11 +22,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(Configuration)' == 'Release'">
-	  <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
```
Comment (drwill-ms) [inline iothub/device/samples/getting started/FileUploadSample/FileUploadSample.csproj:25]: For some reason the tabs were causing reference failures for me.

File: iothub/device/samples/getting started/TwinSample/TwinSample.csproj [hunk 3]
```diff
@@ -14,11 +14,12 @@
...
 
   <ItemGroup Condition="'$(Configuration)' == 'Release'">
-	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(Configuration)' == 'Debug'">
-	<ProjectReference Include="..\..\..\src\Microsoft.Azure.Devices.Client.csproj" />
+    <ProjectReference Include="..\..\..\src\Microsoft.Azure.Devices.Client.csproj" />
+    <ProjectReference Include="..\..\..\..\..\shared\src\Microsoft.Azure.Devices.Shared.csproj" />
```
Comment (drwill-ms) [inline iothub/device/samples/getting started/TwinSample/TwinSample.csproj:22]: Needed a project reference to shared. Not sure why.

File: iothub/device/samples/solutions/PnpDeviceSamples/PnpConvention/PnpHelpers.csproj [hunk 4]
```diff
@@ -18,4 +17,12 @@
     <PackageReference Include="DotNetty.Common" Version="0.7.4" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(Configuration)' == 'Release'">
```
Comment (drwill-ms) [inline iothub/device/samples/solutions/PnpDeviceSamples/PnpConvention/PnpHelpers.csproj:20]: This one didn't have the release/debug logic references

**Review by drwill-ms — COMMENTED** [review]
Ready for review.

**Review by andyk-ms — APPROVED** [review]

**Pull request merged**